.prettierrc.cjs

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  plugins: ["prettier-plugin-tailwindcss"],
+};
```

Comment:
This file was a huge pain, just like the last time I did this.

You'll find suggestions along the lines of 

```js
// .prettierrc.js or prettier.config.js
module.exports = {
  plugins: [require('prettier-plugin-tailwindcss')],
}
```

But I haven't been able to get this to work.

See: https://github.com/pnpm/pnpm/issues/4700#issuecomment-1121834982

---

Using the string array my code above works with `.prettierrc.cjs` and `prettier.config.cjs` files (`module.exports` syntax). 

If you want to use `export default` syntax, you'll have to use  `mjs` files.

Now it's working with both VS Code Format on Save and the Prettier CLI.

Reply:
Anyways, I'll need to stash this somewhere so I can dig it out again in the future. Have a feeling it will pop up again.